ecs-agent/api/ecs/statechange.go

```diff
@@ -124,6 +126,9 @@ func (c *ContainerStateChange) String() string {
 // String returns a human readable string representation of a TaskStateChange.
 func (change *TaskStateChange) String() string {
```

Comment:
nit: Can we update task state change string unit testing to account for case where` ClusterARN` is populated?